src/styles/blocks/page.scss

```diff
@@ -0,0 +1,71 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  font-size: 16px;
+  line-height: 27px;
+  color: $page-text-grey-color;
+  scroll-behavior: smooth;
+
+  &__body {
+    &--with-menu {
+      overflow: hidden;
+    }
+  }
+
+  &__menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+
+    transform: translateX(-100%);
+    transition: transform $transition-duration;
+
+    &:target {
+      transform: translateX(0);
+
+    }
+  }
+
+  &__section {
+    margin-bottom: 64px;
+
+    @include onDesktop {
+      margin-bottom: 104px;
+    }
+  }
+
+  &__section-title {
+    font-weight: bold;
+    font-size: 36px;
+    line-height: 36px;
+    text-align: center;
+    letter-spacing: -1.5px;
+    color: $black-color;
+    margin-bottom: 32px;
+
+    @include onDesktop {
+      margin-bottom: 56px;
+      font-size: 48px;
+      line-height: 48px;
+    }
+
+    &::after {
+      content: "";
+      display: block;
+      width: 23px;
+      height: 16px;
+      margin: 4px auto 0;
+
+      background: center / contain no-repeat url(../images/crown.svg);
+    }
+  }
+
+  &__footer {
+    height: 206px;
```

Comment:
it's better not to hardcode such numbers, but to use variables for them

Reply:
even if this very height (206px) is used only once?

Reply:
yes. It's not a strict rule, but it's recommended